vhdbuilder/packer/list-images.sh

```diff
@@ -3,9 +3,31 @@ set -euxo pipefail
 
 CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-docker}"
 
+function generate_image_bom_for_containerd() {
+    temp_image_bom=/opt/azure/containers/temp-image-bom.json
+    IFS_backup=$IFS
+    IFS=$'\n'
+    ctr_list=$(ctr --namespace k8s.io image list | sed 1d | awk '{print $1, $3}')
+    crictl_list=$(crictl images --no-trunc | sed 1d | awk '{printf "%s:%s %s\n", $1,$2,$3;}')
+
+    for image in $ctr_list; do
+        tag=$(echo $image | awk '{print $1}')
+        # skip image tags that consist of a sha256 hash
+        [[ $tag == sha256* ]] && continue
+        digest=$(echo $image | awk '{print $2}')
+        id=$(echo "$crictl_list" | grep -e "$tag" | awk '{print $2}')
```

Comment:
similar q above, if ctr has the tag, confused why we need both

Reply:
ctr has both the tag and the digest sha, but not the id sha - that comes from crictl

Reply:
I was wrong! all can come from ctr, now simplified